src/gui/effects/qgseffectstackpropertieswidget.cpp

```diff
@@ -479,13 +471,24 @@ void QgsEffectStackCompactWidget::showDialog()
   QgsPanelWidget *panel = QgsPanelWidget::findParentPanel( qobject_cast< QWidget * >( parent() ) );
   if ( panel && panel->dockMode() )
   {
+<<<<<<< HEAD
```

Comment:
@3nids , I've seen the qgis backport bot creating these problematic PRs a couple of times already. What's going on here?

Reply:
That's just the merge conflict formatting but it seems that the bot just keeps the formatting and doesn't resolve the conflicts. :/